Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -8,6 +8,11 @@
   NODE_MAP = create_node_map()
   # -*- mode: Swift -*-
   # Ignore the following admonition it applies to the resulting .swift file only
+
+  def get_expressible_as_conformances(conforming_type):
+    expressible_as_protocols = (SYNTAX_COLLECTION_EXPRESSIBLE_AS_CONFORMANCES.get(conforming_type) or [])[:]
```

Comment:
Could you add a comment why `[:]` is necessary here? Something like
```
# Use [:] to make sure we copy the conformances so that we don't modify `SYNTAX_COLLECTION_EXPRESSIBLE_AS_CONFORMANCES` when adding elements to `expressible_as_protocols` below.
```